compiler/arguments/src/org/jetbrains/kotlin/arguments/description/CommonKlibBasedCompilerArguments.kt

```diff
@@ -107,4 +108,16 @@ The only observable effect is that a custom ABI version is written to KLIB manif
             introducedVersion = KotlinReleaseVersion.v2_2_0,
         )
     }
+
+    compilerArgument {
+        name = "Xklib-zip-file-accessor-cache-limit"
+        description = "Size of cache to be used for the klib zip file accessor. Default is 0.".asReleaseDependent()
+        valueType = IntType(
+            defaultValue = 0.asReleaseDependent()
+        )
+
+        lifecycle(
+            introducedVersion = KotlinReleaseVersion.v2_1_21
```

Comment:
New CLI parameter can only be introduced since 2_3_0. Not `KotlinReleaseVersion.v2_1_21`.